portals/devportal/src/main/webapp/source/src/app/components/Apis/Listing/APICategoryThumb.jsx

```diff
@@ -69,20 +84,23 @@ function APICategoryThumb(props) {
     }
     return (
         <StyledLink to={categoryLink} className={classes.textWrapper}>
-            <Tooltip placement='right' title={category.description.length <= 50 ? '' : category.description}>
-                <ListItem button alignItems='flex-start'>
+            <Tooltip placement='top' title={category.description}>
+                <ListItemButton
+                    className={classNames(classes.listItemButton, { [classes.selectedCategory]: selected })}
+                    onClick={props.onClick}
```

Comment:
Is there an onClick being passed down through props? If so, shall we add that to the prop deconstruction at line 78? Also, this onClick is missing the prop validation:

```
APICategoryThumb.propTypes = {
    classes: PropTypes.shape({}).isRequired,
    category: PropTypes.shape({}).isRequired,
    path: PropTypes.shape({}).isRequired,
    selected: PropTypes.bool,
    + let's add prop validation here for onClick
};
```